packages/blade/src/components/Typography/Display/Display.stories.tsx

```diff
@@ -0,0 +1,145 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import { Title as StorybookTitle } from '@storybook/addon-docs';
+import type { ReactElement } from 'react';
+import { Text } from '../Text';
+import { Code } from '../Code';
+import type { DisplayProps } from './';
+import { Display as DisplayComponent } from './';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import { getStyledPropsArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import { Box } from '~components/Box';
+import { List, ListItem } from '~components/List';
+import { isReactNative } from '~utils';
+
+const Page = (): ReactElement => {
+  return (
+    <StoryPageWrapper
+      componentDescription="The Display component adds a strong visual touch. Utilize it to create eye-catching sections on your landing pages."
+      componentName="Display"
+      figmaURL={{
+        paymentTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=13708%3A180902&mode=design&t=1sYKFDyXDXlugu2m-1',
+        bankingTheme:
+          'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?type=design&node-id=10344%3A189840&mode=design&t=Np5DsiaBhYmDNpTw-1',
+      }}
+    >
+      <StorybookTitle>Usage</StorybookTitle>
+      <Sandbox>
+        {`
+          import { Display } from '@razorpay/blade/components';
+
+          function App(): React.ReactElement {
+            return (
+              <Display size="large">Blade by Razorpay</Display>
+            )
+          }
+
+          export default App;
+        `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+const DisplayStoryMeta: Meta<DisplayProps> = {
+  title: 'Components/Typography/Display',
+  component: DisplayComponent,
+  args: {
+    size: 'small',
+    type: 'normal',
+    children: 'Power your finance, grow your business',
+    contrast: 'low',
+    as: undefined,
+  },
+  argTypes: {
```

Comment:
once you select the color prop, it doesn't have any effect if you try to change the type prop. try selecting the color as `primary.300` for example and then try to change the `type` prop

Reply:
@kamleshchandnani I believe this is intentional for all typography elements as we have these type tokens for surface text only. Otherwise, it's a bug which we will need to fix for all typography elements.

<img width="1099" alt="Screenshot 2023-09-04 at 7 16 25 AM" src="https://github.com/razorpay/blade/assets/46647141/8cb03e00-23cf-41a1-999d-6e8df0ac3845">